### PR TITLE
MAKES sub-graph tutorial feedback update 2

### PR DESCRIPTION
**Pull request by alch-msft**
1. Algin params between MagDataToMakesJson.usql script and how-to-create-index-from-mag.md tutorial
2. Add scape for URLs in bold 
3. Update MakesIndexResourceConfigFilePath mispell in tutorial and reference
4. Update trouble shooting guide to include azure quota issue and batch service not being registered issue.

### CODE REVIEW

File: academic-services/knowledge-exploration-service/MagDataToMakesJson.usql [hunk 1]
```diff
@@ -23,7 +23,10 @@
     
     // The storage output path where the JSON files will be created
     // The if the output destination type is Azure storage (output path starts with "wasb"), the container for the output location needs to be created before running the script
-    DECLARE EXTERNAL @Out_OutputPath string = "";  // Example: "wasb://makessubgraph@mymakesstore/mag-2020-02-07/microsoft"
+    DECLARE EXTERNAL @Out_OutputPath string = "";  // Example: "wasb://makessubgraph@mymakesstore/2020-02-07/microsoft/entities"
```
Comment (RickRogahn) [inline academic-services/knowledge-exploration-service/MagDataToMakesJson.usql:26]: update the date of this document as well in the meta-data

File: academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md [hunk 2]
```diff
@@ -90,13 +90,16 @@ If you have not done so already, download the Kesm.exe tool from your MAKES subs
     | Values | Description | Example |
     |---------|---------|------|
     |**`<IndexResourceName>`** | The name of the indexing resources. This step will create a Resource Group, Azure Batch account, and an Azure Storage Account using this name. | **makesindexres** |
-    |**`<MakesPackageLocation>`** | The base URL to a MAKES release. The indexer, preprocessor, and jobManager package inside the release will be used to set up the Azure Batch account|  **https://mymakesstore.blob.core.windows.net/makes/2020-02-07/** |
+    |**`<MakesPackageLocation>`** | The base URL to a MAKES release. The indexer, preprocessor, and jobManager package inside the release will be used to set up the Azure Batch account|  **https&#58;//mymakesstore.blob.core.windows.net/makes/2020-02-07/** |
```
Comment (RickRogahn) [inline academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md:93]: ", and jobManager package**s**

File: academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md [hunk 3]
```diff
@@ -90,13 +90,16 @@ If you have not done so already, download the Kesm.exe tool from your MAKES subs
...
     |**`<MakesIndexResourceConfigFilePath>`** | The local output file for saving the indexing resource configuration information to build your index. | **makesIndexResConfig.json** |
 
 1. Run the command
     > [!NOTE]
     > If have multiple Azure Subscriptions and/or Tenants, you'll specify additional parameters.
     > See [Azure login failure due to multiple subscriptions or multiple tenants being tied to a single Azure account](resources-troubleshoot-guide.md#Azure-login-failure-due-to-multiple-subscriptions-or-multiple-tenants-being-tied-to-a-single-Azure-account) for more details.
+    >
+    > If your azure subscription hasn't register for Azure Batch service, you'll need to do so.
```
Comment (RickRogahn) [inline academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md:101]: capitalize Azure, also "hasn't been registered"

File: academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md [hunk 4]
```diff
@@ -127,6 +130,9 @@ The final step to generate your index is to submit a build index job to Azure Ba
 > Generating indexes can cost money.
 >
 > Time to create an index can be up to 3 hours.
+>
+> Due to COVID-19, new Azure Batch Accounts in many region will have 0 vCore limit. This will cause index jobs to get stuck.
```
Comment (RickRogahn) [inline academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md:134]: Replace "Due to COVID-19," with "Due to regional restrictions,"

File: academic-services/knowledge-exploration-service/resources-troubleshoot-guide.md [hunk 5]
```diff
@@ -2,24 +2,30 @@
...
 
 ## Azure login failure due to multiple subscriptions or multiple tenants being tied to a single Azure account
 
-If you have multiple azure tenants associated with a single azure account, you'll need to specify the Azure Active Directory directory domain name associated with the Azure Subscription that you would like to use to execute the command. (e.g. "--AzureActiveDirectoryDomainName constco.onmicrosoft.com").  
+If you have multiple Azure tenants associated with a single Azure account, you'll need to specify the Azure Active Directory directory domain name associated with the Azure Subscription that you would like to use to execute the command. (e.g. "--AzureActiveDirectoryDomainName constco.onmicrosoft.com").  
 
-You can find this information by logging into azure portal, searching for "Azure Active Directory" and viewing resource detail page. The domain name to use will be in the "Overview" section at the top left.
+You can find this information by logging into Azure Portal, searching for "Azure Active Directory" and viewing resource detail page. The domain name to use will be in the "Overview" section at the top left.
```
Comment (RickRogahn) [inline academic-services/knowledge-exploration-service/resources-troubleshoot-guide.md:14]: "You can find this information by logging into **the** Azure Portal, searching for 'Azure Active Directory' and viewing **the** resource detail page."

File: academic-services/knowledge-exploration-service/resources-troubleshoot-guide.md [hunk 6]
```diff
@@ -2,24 +2,30 @@
...
 
 If you have multiple subscriptions associated with the same Azure Active Directory doamin and want to use the non-default subscription, you'll need to specify the subscription Id associated with the subscription to execute the command. (e.g. "--AzureSubscriptionId XXXXXXX-XXXX-XXX-XXXX-XXXXXXX")
 
-You can find this information by logging into azure portal, searching for "Subscriptions" and viewing resource detail page. The subscription Id will be listed in the table.
+You can find this information by logging into Azure Portal, searching for **Subscriptions** and viewing resource detail page. The subscription Id will be listed in the table.
```
Comment (RickRogahn) [inline academic-services/knowledge-exploration-service/resources-troubleshoot-guide.md:18]: "You can find this information by logging into **the** Azure Portal, searching for **Subscriptions** and viewing **the** resource detail page"

File: academic-services/knowledge-exploration-service/resources-troubleshoot-guide.md [hunk 7]
```diff
@@ -2,24 +2,30 @@
...
 
-## Azure subscription limit
+## Azure Subscription quota limit reached
 
-If you encounter errors such as "operation could not be completed as it results in exceeding approved XXXXXX quota", you'll either need to contact azure support to increase your account quota or change the MAKES host machine settings.  
+If you encounter errors such as "operation could not be completed as it results in exceeding approved XXXXXX quota", you'll need to contact Azure support to increase your account quota, try a different region where you have more Azure quota, or change the MAKES host machine sku/worker machine sku settings. For most Microsoft Academic sub-graph index builds, you can scale down to **Standard_D8_v3** sku to build and host the indexes.
+
+## Azure Batch service not registered
+
+If you encouter errors such as "The subscription registration is in 'Unregistered' state. The subscription must be registered to use namespace 'Microsoft.Batch'....", you'll need to register your subscription to use Azure Batch service.
```
Comment (RickRogahn) [inline academic-services/knowledge-exploration-service/resources-troubleshoot-guide.md:26]: "If you encouter errors such as "The subscription registration is in 'Unregistered' state. The subscription must be registered to use **the** namespace 'Microsoft.Batch'..."

File: academic-services/knowledge-exploration-service/resources-troubleshoot-guide.md [hunk 8]
```diff
@@ -2,24 +2,30 @@
...
+
+You can use Azure Portal to register Azure Batch service for your subscription. To do this, log into Azure Portal, search for **Subscriptions** and select the subscription you want to register Azure Batch service for. Then, select **Resource providers** and search for **Microsoft.Batch**. Finally, select **Microsoft.Batch** from the provider list and select **register**. This should only take a few minutes.
```
Comment (RickRogahn) [inline academic-services/knowledge-exploration-service/resources-troubleshoot-guide.md:28]: "You can use **the** Azure Portal to register Azure Batch service for your subscription. To do this, log into **the** Azure Portal, search for **Subscriptions** and select the subscription you want to register Azure Batch service **under**. Then, select **Resource providers** and search for **Microsoft.Batch**. Finally, select **Microsoft.Batch** from the provider list and select **register**. This should only take a few minutes."

File: academic-services/knowledge-exploration-service/resources-troubleshoot-guide.md (hunk 7 above)
Comment (alch-msft) [reply academic-services/knowledge-exploration-service/resources-troubleshoot-guide.md:26]: copied error message word for wrod.

**Pull request merged**